client/state/happiness-engineers/selectors.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+function getHappinessEngineersState( state ) {
+	return state.happinessEngineers;
+}
+
+function createHappinessEngineersSelector( fn ) {
+	return createSelector( fn, ( state ) => [ getHappinessEngineersState( state ) ] );
+}
+
+/**
+ * Returns happiness engineers
+ * @returns [] happiness engineers
+ */
+export const getHappinessEngineers = createHappinessEngineersSelector(
+	( state ) => getHappinessEngineersState( state ).items
```

Comment:
This doesn't seem like something we need to memoize, there's very little computation happening.